test/spec/features/popup-menu/PopupMenu.example-entries.json

```
@@ -0,0 +1,1487 @@
+[
+  {
+    "id": "create-exclusive-gateway",
+    "label": "Exclusive gateway",
+    "group": {
+      "id": "gateways",
+      "name": "Gateways"
+    }
+  },
+  {
+    "id": "create-parallel-gateway",
+    "label": "Parallel gateway",
+    "group": {
+      "id": "gateways",
+      "name": "Gateways"
+    }
+  },
+  {
+    "id": "create-inclusive-gateway",
+    "label": "Inclusive gateway",
+    "group": {
+      "id": "gateways",
+      "name": "Gateways"
+    },
+    "search": "or",
+    "rank": -1
+  },
+  {
+    "id": "create-complex-gateway",
+    "label": "Complex gateway",
+    "group": {
+      "id": "gateways",
+      "name": "Gateways"
+    },
+    "rank": -1
+  },
+  {
+    "id": "create-event-based-gateway",
+    "label": "Event-based gateway",
+    "group": {
+      "id": "gateways",
+      "name": "Gateways"
+    }
+  },
+  {
+    "id": "create-task",
+    "label": "Task",
+    "group": {
+      "id": "tasks",
+      "name": "Tasks"
+    }
+  },
+  {
+    "id": "create-user-task",
+    "label": "User task",
+    "group": {
+      "id": "tasks",
+      "name": "Tasks"
+    },
+    "search": "form human"
```

Shouldn't it be this, going into the future? 

```suggestion
    "search": [ "form", "human" ]
```

---

we could adjust that in the source, but its not coming from element templates